src/main/java/service/operation/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package service.operation;
+
+import java.util.Map;
+import model.FruitTransaction;
+
+public class OperationStrategyImpl implements OperationStrategy {
+
+    private final Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;
+
+    public OperationStrategyImpl(Map<FruitTransaction.Operation,
+            OperationHandler> operationHandlers) {
+        this.operationHandlerMap = operationHandlers;
+    }
+
+    @Override
+    public OperationHandler get(FruitTransaction.Operation operation) {
```

Comment:
Ensure that the get method handles the case where the operation is not found in the map. This can prevent potential `NullPointerException` when the method returns null. Consider throwing a specific exception if the operation is not supported.